### PR TITLE
RELEASE.md notes for branch builds

### PR DESCRIPTION
**Pull request by chrisplo**

### CODE REVIEW

**Comment by chrisplo** [comment]
Example:
```
NETPLUGIN_BRANCH=v2plugin_local_versioned NETPLUGIN_OWNER=chrisplo make build
```

```
$ ls -l contiv-devbuild/contiv_cache/netplugin-*
-rw-r--r--  1 chrisplo  staff  20887462 Nov  6 11:15 contiv-devbuild/contiv_cache/netplugin-ca1b582.tar.bz2
lrwxr-xr-x  1 chrisplo  staff        25 Nov  6 11:14 contiv-devbuild/contiv_cache/netplugin-chrisplo-v2plugin_local_versioned.tar.bz2 -> netplugin-ca1b582.tar.bz2
```

File: RELEASE.md [hunk 1]
```diff
@@ -1,6 +1,16 @@
 # Automated releases
 TBD
 
+# Testing installer builds of any netplugin branch
+To create a full installer for legacy docker swarm of any branch of contiv, additionally set these two environment variables:
+
+* NETPLUGIN_BRANCH - the branch to compile instead of latest release
```
Comment (dseevr) [inline RELEASE.md]: Isn't this "branch to compile instead of current branch"?

Comment (chrisplo) [reply RELEASE.md]: if this is not set, install pulls release versions only based on a checked in variable value or environment override, which is unrelated to any workspace branch

Comment (dseevr) [reply RELEASE.md]: Could the description be something more like that?  The current text didn't imply any of that to me as someone reading it for the first time

Comment (chrisplo) [reply RELEASE.md]: done

**Comment by kahou82** [comment]
👍

**Pull request merged**